app/webpacker/components/Tickets/TicketWorkbenches/EditPersonActionerView/OldDataSyncInfo.jsx

```diff
@@ -0,0 +1,56 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import React, { useMemo } from 'react';
+import { Button, Message } from 'semantic-ui-react';
+import syncEditPersonRequest from '../../api/editPerson/syncEditPersonRequest';
+import Loading from '../../../Requests/Loading';
+import Errored from '../../../Requests/Errored';
+
+export default function OldDataSyncInfo({ ticketDetails, currentStakeholder }) {
+  const {
+    ticket: {
+      id,
+      metadata: {
+        tickets_edit_person_fields: ticketsEditPersonFields,
+        person,
+      },
+    },
+  } = ticketDetails;
+
+  const queryClient = useQueryClient();
+  const {
+    mutate: syncEditPersonRequestMutate,
+    isPending,
+    isError,
+    error,
+  } = useMutation({
+    mutationFn: syncEditPersonRequest,
+    onSuccess: () => {
+      // We've invalidated the ticket details because we can't confirm that the updated value
+      // matches the current person's data. To be safe, we're refetching the information, as
+      // the person's details may have changed since the last time they were accessed.
+      queryClient.invalidateQueries(['ticket-details', id]);
+    },
```

Comment:
You can send the serialized ticket (in its current, brand-new updated form) as a success response from the `sync_edit_person_request` controller method. Then you can use `setQueryData` instead of `invalidateQueries` here, which saves you one network request 👍

Reply:
Makes sense, done.